README.md

```diff
@@ -25,6 +25,8 @@ Content:
       Supported options are `default`, `gcp`, `aws`, and `openstack`.
     * `SOPS fingerprint` is the gpg fingerprint that will be used for SOPS encryption.
       You need to set this or the environment variable `CK8S_PGP_FP` the first time SOPS is used in your specified config path.
+    * Additionally, the init script will copy your public SSH key from `~/.ssh/id_rsa.pub` to the config path.
+      If you want a different SSH key file copied, set the environment variable `SSH_KEY_LOCATION` to the absolute path of the file. The file can contain several SSH keys, one per line.
```

Comment:
I'm not a fan of mixing envs and positional arguments. Is there any particular reason for why we settle on env var for this? I also think that we shouldn't make this optional and default to `~/.ssh/id_rsa.pub`, better to have the user explicitly choose. Then you're less prone to accidentally only include your own ssh key.

Thoughts on this?

Reply:
The init script doesn't really allow for several optional arguments at the moment (it just checks for the SOPS fingerprint in a specific position) and I wasn't sure if this was the moment to change it into something that works better (i.e. specifying flags like `-sops=... -ssh=...`). But as you say, maybe we want to make it mandatory anyway, the only thing I don't like about that is the case where the user just wants to use their `~/.ssh/id_rsa.pub` and they still have to specify the path to it - feels a bit redundant.